roles/edpm_libvirt/tasks/configure.yml

```diff
@@ -115,14 +115,16 @@
     - libvirt
   become: true
   loop:
-    - {"src": "/var/lib/openstack/certs/libvirt/tls.crt", "dest": "/etc/pki/libvirt/tls.crt"}
-    - {"src": "/var/lib/openstack/certs/libvirt/tls.key", "dest": "/etc/pki/libvirt/private/tls.key"}
-    - {"src": "/var/lib/openstack/cacerts/libvirt/tls-ca-bundle.pem", "dest": "/etc/pki/CA/libvirt/tls-ca-bundle.pem"}
+    - {"src": "{{ edpm_libvirt_tls_cert_src_dir }}/tls.crt", "dest": "/etc/pki/libvirt/servercert.pem"}
+    - {"src": "{{ edpm_libvirt_tls_cert_src_dir }}/tls.key", "dest": "/etc/pki/libvirt/private/serverkey.pem"}
+    - {"src": "{{ edpm_libvirt_tls_cert_src_dir }}/tls.crt", "dest": "/etc/pki/libvirt/clientcert.pem"}
+    - {"src": "{{ edpm_libvirt_tls_cert_src_dir }}/tls.key", "dest": "/etc/pki/libvirt/private/clientkey.pem", "mode": "0644"}
```

Comment:
0644 doesn't look right for the key, should that have been set on the cacert?

Reply:
If the intention is to allow access to non-root users (is that wise?) then 0640 and a specific group perhaps

Reply:
to the client key yes. because no one shoudl be logging in to the compute ndoes as root and i want ti work for the non admin user by default
with that said i could make it 0640 and change the ownership to root:libvirt

that woudl mean only members of the libvirt group could use it.

the virt-pki-validate script is actully what proped me to change it 

https://github.com/libvirt/libvirt/blob/11839c8cc1d204ec3abdb3a0d9499b64dc83fc83/tools/virt-pki-validate.in#L230

 if [ "$MOD" != "-rw-r--r--" ]
            then
                echo The client private key need to be read by client tools
                echo "as root do: chmod 644 $LIBVIRTP/clientkey.pem"
            fi

so they recommend using 644 but that because they do not have a libvirt group by default.
we do so we can make it tigher if we want too.

Reply:
i guess it would work with sudo without this and the user you will conenct in as to debug should have that so i can likely revert to 600 ill need to test that but thats not unreasonabel to do.